src/views/Miscellaneous.vue

```diff
@@ -0,0 +1,145 @@
+        </aside>
+      </main>
+    </ion-content>
+  </ion-page>      
+</template>
+
+<script lang="ts">
+import { DateTime } from 'luxon';
+import {
+  IonContent,
+  IonHeader,
+  IonInfiniteScroll,
+  IonInfiniteScrollContent,
+  IonItem,
+  IonLabel,
+  IonList,
+  IonListHeader,
+  IonMenuButton,
+  IonNote,
+  IonPage,
+  IonTitle,
+  IonToolbar,
+  isPlatform,
+} from '@ionic/vue';
+import { defineComponent } from 'vue';
+import { useRouter } from 'vue-router'
+import { mapGetters, useStore } from 'vuex'
+import emitter from '@/event-bus';
+import JobConfiguration from '@/components/JobConfiguration.vue';
+import { chevronForwardOutline } from "ionicons/icons";
+
+export default defineComponent({
+  name: 'Miscellaneous',
+  components: {
+    IonContent,
+    IonHeader,
+    IonInfiniteScroll,
+    IonInfiniteScrollContent,
+    IonItem,
+    IonLabel,
+    IonList,
+    IonListHeader,
+    IonMenuButton,
+    IonNote,
+    IonPage,
+    IonTitle,
+    IonToolbar,
+    JobConfiguration
+  },
+  mounted() {
+    this.getMiscellaneousJobs();
+  },
+  data() {
+    return {
+      title: '',
+      currentJobStatus: '',
+      isJobDetailAnimationCompleted: false,
+      isDesktop: isPlatform('desktop')
+    }
+  },
+  computed: {
+    ...mapGetters({
+      miscellaneousJobs: 'job/getMiscellaneousJobs',
+      getCurrentEComStore:'user/getCurrentEComStore',
+      currentJob: 'job/getCurrentJob',
+      isMiscellaneousJobsScrollable: 'job/isMiscellaneousJobsScrollable'
+    })
+  },
+  methods: {
+    async viewJobConfiguration(job: any) {
+      this.title = job.jobName
+      this.currentJobStatus = job.tempExprId
+
+      await this.store.dispatch('job/updateCurrentJob', { job });
+      if(!this.isDesktop && job?.jobId) {
+        this.router.push({name: 'JobDetails', params: { title: this.title, jobId: job?.jobId, category: "miscellaneous"}});
+        return;
+      }
+
+      if (job && !this.isJobDetailAnimationCompleted) {
+        emitter.emit('playAnimation');
+        this.isJobDetailAnimationCompleted = true;
+      }
+    },
+    async getMiscellaneousJobs(viewSize = 20, viewIndex = '0') {
+      await this.store.dispatch('job/fetchMiscellaneousJobs', {eComStoreId: this.getCurrentEComStore.productStoreId, viewSize, viewIndex});
+    },
+    async loadMoreMiscellaneousJobs (event: any) {
+      this.getMiscellaneousJobs(
+        undefined,
+        Math.ceil(this.miscellaneousJobs.length / (process.env.VUE_APP_VIEW_SIZE as any)).toString()
+      ).then(() => {
+        event.target.complete();
+      })
+    },
+    timeFromNow (time: any) {
+      const timeDiff = DateTime.fromMillis(time).diff(DateTime.local());
+      return DateTime.local().plus(timeDiff).toRelative();
+    },
+  },
+  setup() {
+    const store = useStore();
+    const router = useRouter();
+
+    return {
+      chevronForwardOutline,
+      router,
+      store,
+    };
+  },
+})
+</script>
+
+<style scoped>
+main > section {
```

Comment:
indentation.

Also is this css not global? I don't think we need to re-add this for this page.

Reply:
Actually sir, that global css was getting applied but was not visible, then I discussed with Azkya ma'am and decided to override it with local css.
Fixed indetation.